src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy

```diff
@@ -14,7 +14,7 @@ class InspectContainerResponseHandler implements ResponseHandler<String, Object>
         logger.quiet "Links       : $container.hostConfig.links"
         logger.quiet "Volumes     : $container.volumes"
         logger.quiet "VolumesFrom : $container.hostConfig.volumesFrom"
-        String exposedPorts = container.config?.@exposedPorts ? container.config.exposedPorts : '[]'
+        String exposedPorts = container.config.exposedPorts ? container.config.exposedPorts.toString() : '[]'
```

Comment:
This fix we talked about a couple weeks ago was merged. So a win for simplicity here.